pkg/webclient/cloudproviders/cloudproviders.go

```diff
@@ -50,3 +51,61 @@ func (c *CloudProvidersClient) GetCloudProviderByName(ctx context.Context, name
 		KeystoresCount: cp.GetKeystoresCount(),
 	}, nil
 }
+
+func (c *CloudProvidersClient) GetCloudKeystore(ctx context.Context, cloudKeystoreID *string, cloudKeystoreName *string, cloudProviderID *string, cloudProviderName *string) (*domain.CloudKeystore, error) {
+
+	if cloudKeystoreID == nil {
+		if cloudKeystoreName == nil || (cloudProviderID == nil && cloudProviderName == nil) {
+			return nil, fmt.Errorf("following are accepted for provisioning: keystore ID, or both keystore Name and provider Name, or both keystore Name and provider ID")
```

Comment:
While the three combinations are possible, users don't need to know that a combination of ID and name is available. 
If the user can get the name of the keystone, they can also get the provider's name.

In that sense, we should only advise two scenarios:
- keystore ID or
- provider name and keystore name

Reply:
I mean, to that sense in the webclient, the endpoint provides the ability to set the Cloud Provider ID. I was not constraining it in there since I was already doing so in the Provisioning interface method. Do you think it also applies for in there?

Reply:
In that case, constrain it. They dont need all options available. Either keystore ID or provider/keystore names

Reply:
Done by latest changes